### PR TITLE
chore: add project metadata to package.json

### PR DESCRIPTION
**Pull request by NachoVazquez**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

The project package.json doesn't contain the necessary metadata that npm would read. Additionally, the logo is local and therefore not accessible on npm.

## What is the new behavior?

We have included on the package.json the following metadata:

- description
- homepage
- license
- repository
- keywords
- contributors
- bugs

The lumberjack logo was moved to a CDN.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 > Chop and cut Angular logs like a professional lumberjack.
 
 <p align="center">
- <img width="40%" height="40%" src="./logo.svg">
+ <img width="40%" height="40%" src="https://pub-2294738bc2c249ff8040505bf960c018.r2.dev/logo.svg">
```
Comment (LayZeeDK) [inline README.md:4]: question: What's this host?

File: packages/ngworker/lumberjack/package.json [hunk 2]
```diff
@@ -1,6 +1,41 @@
 {
   "name": "@ngworker/lumberjack",
   "version": "14.0.0",
+  "description": "Lumberjack is a versatile Angular logging library, specifically designed to be extended and customized",
+  "homepage": "https://github.com/ngworker/lumberjack#readme",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ngworker/lumberjack.git",
+    "directory": "packages/ngworker/lumberjack"
```
Comment (LayZeeDK) [inline packages/ngworker/lumberjack/package.json:10]: question: Where can I read more about this piece of metadata?

File: README.md (hunk 1 above)
Comment (NachoVazquez) [reply README.md:4]: Cloudflare R2

File: packages/ngworker/lumberjack/package.json (hunk 2 above)
Comment (NachoVazquez) [reply packages/ngworker/lumberjack/package.json:10]: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository

It is used for monorepos

**Review by santoshyadavdev — APPROVED** [review]
Looks good to me 😊 adding a directory property is a great idea

**Pull request merged**